### PR TITLE
Schema V1-2-0

### PR DESCRIPTION
**Pull request by lyonsja**

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -113,6 +114,7 @@
           "type": "string",
           "title": "Update-Path Schema",
           "description": "Folder where the application expects auto-updated OTA packages to be found. Default is value specified in 'target path'.",
+          "default": "",
```
Comment (codybrookshear) [inline schema.json]: Is there any json-schema way to actually tie this value to target-path as the `description` implies?

File: schema.json [hunk 2]
```diff
@@ -95,6 +94,7 @@
           "type": "string",
           "title": "Target-Path Schema",
           "description": "Base destination folder where any archive/fill will be inflated/written. Default is device internal SD-card root",
+          "default": "",
```
Comment (codybrookshear) [inline schema.json]: Is there any json-schema way to actually tie this value to target-path as the description implies?

File: schema.json [hunk 3]
```diff
@@ -104,6 +104,7 @@
           "type": "string",
           "title": "Install-Path Schema",
           "description": "Folder where the application expects auto-installed/auto-updated APKs to be found. Default is value specified in 'target path'.",
+          "default": "",
```
Comment (codybrookshear) [inline schema.json]: Is there any json-schema way to actually tie this value to target-path as the description implies?

File: schema.json (hunk 1 above)
Comment (lyonsja) [reply schema.json]: I am setting the value for each of these paths using the JsonHandler class I created.
        These are the relevant calls in ApplyState.
```Java
        _targetPath = new File(JsonHandler.getOptString("global", "target-path"));
        _installPath = new File(JsonHandler.getOptString("global", "install-path"));
        _updatePath = new File(JsonHandler.getOptString("global", "update-path"));
```
If there's no value specified, the default should be used from the schema.

I made the change to the description in the readme--where it no longer sets the other defaults from target-path. However, I missed updating the description in the schema.

Comment (lyonsja) [reply schema.json]: Updated these descriptions in [66ee14f](https://github.com/datalogic/scan2deploy-android-schema/pull/1/commits/66ee14fd43b7b5908d255e246018270e8b5f3bf9).

Comment (lyonsja) [reply schema.json]: After IRL discussion, I have reverted some of these changes and added a definitions section to the schema allowing target-path, install-path, and update-path to all inherit from a base-path value. [bd99240](https://github.com/datalogic/scan2deploy-android-schema/pull/1/commits/bd99240d56838f6e6b003232a4d7409da4d58259)

**Review by codybrookshear — APPROVED** [review]

**Pull request merged**